apps/versions.md

```diff
@@ -1,13 +1,18 @@
 ---
 layout: page
 key: apps-version
-title: Versions
+title: API version
 ---
 
-## Current version
-
-By default, all requests receive the **v1** version of the API. We encourage you to explicitly request this version via the `Accept` header.
+By default, all requests receive the **v1** version of the API.
+We encourage you to explicitly request **v1** via the [`Accept` header](page:apps-media-type), e.g.
 
 {% highlight text %}
 Accept: application/vnd.epages.v1+json
 {% endhighlight %}
+
+or
+
+{% highlight text %}
+Accept: application/vnd.epages.v1+csv
```

Comment:
I think this is not the correct place to mention that. 
If we have to rethink the "media types" section anyways, we should do it there.
API version is just about saying, which version should be requested.